src/component/TodoForm.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const TodoForm = ({ onChange, onSubmit }) => (
```

Comment:
### Nitpick

This isn't important but as a walking ESLint, I can't help it. It helps with readability when you have two line spaces instead of one after your import statements.